File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/domain/model/TermsAndConditions.java
package com.lattis.ellipse.domain.model;

public class TermsAndConditions {

    private String version;
    private String content;

    public String getVersion() {
        return version;
    }

    public void setVersion(String version) {
        this.version = version;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    @Override
    public String toString() {
        return "TermsAndConditions{" +
                "version='" + version + "'" +
                ", content='" + content + "'}";
    }

}

File: velolabs_repos/lattis_android_v2/domain/src/main/java/com/lattis/domain/models/TermsAndConditions.kt
package com.lattis.domain.models

class TermsAndConditions {

    var version: String? = null
    var content: String? = null

    override fun toString(): String {
        return "TermsAndConditions{" +
                "version='" + version + "'" +
                ", content='" + content + "'}"
    }

}
